alvr/server_openvr/cpp/alvr_server/Controller.cpp

```diff
@@ -74,13 +74,15 @@ vr::EVRInitError Controller::Activate(vr::TrackedDeviceIndex_t unObjectId) {
         vr::VRScalarUnits_NormalizedOneSided
     );
 
-    if (this->device_id == HAND_LEFT_ID) {
+    if (this->device_id == HAND_LEFT_ID || this->device_id == HAND_TRACKER_LEFT_ID) {
```

Comment:
Dunno if it makes sense to fix in this pr, but the naming is pretty confusing

Reply:
I agree it's a bit confusing. HAND_LEFT_ID comes from the openxr prefix `/user/hand/left`, but hand tracking does not use paths.

Reply:
You should probably put a comment how that entire thing with controller switching and whatnot works and why so ppl can actually understand it when reading through the code

Reply:
I'm not sure exactly where to put this comment. The switching logic is a bit all over the server code

Reply:
I'd say people trace up the call stack to understand code, so where the c++ code controller updates is called by rust code.

Reply:
done